telemetry/tracks/class-tracks-event.php

```diff
@@ -161,10 +161,18 @@ protected function process_properties(
 			}
 		}
 
+		// Set VIP app ID if it exists.
+		if ( defined( 'VIP_GO_APP_ID' ) ) {
+			$app_id = constant( 'VIP_GO_APP_ID' );
+			if ( is_integer( $app_id ) && $app_id > 0 ) {
+				$event->vipgo_app = $app_id;
+			}
+		}
+
 		// Set VIP organization if it exists.
 		if ( defined( 'VIP_ORG_ID' ) ) {
 			$org_id = constant( 'VIP_ORG_ID' );
-			if ( is_string( $org_id ) && '' !== $org_id ) {
+			if ( is_integer( $org_id ) && $org_id > 0 ) {
 				$event->vipgo_org = $org_id;
```

Comment:
Similar to the app change, let's rename this:

```suggestion
				$event->vip_org_id = $org_id;
```